CONTRIBUTING.md

```diff
@@ -246,6 +246,17 @@ All donated code requires a license compatible with the Apache Software License
 2.0, and donated code will require a change of copyright to reflect the
 OpenTelemetry Authors.
 
+### Donation process
+
+Broadly, these are the steps the OpenTelemetry Governance and Technical
+Committees follow to handle a prospective donation.
+
+1. Per the above, the donating organization creates a GitHub issue (with the appropriate template) in the `community` repository.
+2. The Technical Committee (TC) will do diligence, write a report, and attach that report to the GitHub issue
```

Comment:
```suggestion
2. The Technical Committee (TC) will do due diligence, write a report, and attach that report to the GitHub issue.
```

Reply:
"do due diligence" translates roughly to "do things that need to be done" which strikes me as a truism. I would prefer something more specific like technical analysis. It should also be committed to the community repo when it is completed.

I also believe there should be a template to follow for that.

Reply:
@dyladan no objection to a diligence template, though arguably that should come from the TC. I'm glad to take a first pass if helpful.

I don't want to spell out the diligence details in this particular doc, though... seems like a better thing for the template ^^^.

Reply:
```suggestion
2. The Technical Committee (TC) will do diligence, write a report, and attach that report to the GitHub issue. Writing the report may require meeting and discussing prospective technology with different vendors in the community and can be a lengthy process. TC member driving the report will post updates to the issue.
```